2010-02-17  Stepan Kasal  <skasal@redhat.com>

	* cmulocal/bsd_sockets.m4: better quoting of macro params

--- cyrus-sasl-2.1.23/cmulocal/bsd_sockets.m4	2005-04-26 21:14:07.000000000 +0200
+++ cyrus-sasl-2.1.23/cmulocal/bsd_sockets.m4	2010-02-17 15:03:34.000000000 +0100
@@ -10,14 +10,14 @@
 	save_LIBS="$LIBS"
 	LIB_SOCKET=""
 	AC_CHECK_FUNC(connect, :,
-		AC_CHECK_LIB(nsl, gethostbyname,
+		[AC_CHECK_LIB(nsl, gethostbyname,
 			     LIB_SOCKET="-lnsl $LIB_SOCKET")
 		AC_CHECK_LIB(socket, connect,
-			     LIB_SOCKET="-lsocket $LIB_SOCKET")
+			     LIB_SOCKET="-lsocket $LIB_SOCKET")]
 	)
 	LIBS="$LIB_SOCKET $save_LIBS"
 	AC_CHECK_FUNC(res_search, :,
-		LIBS="-lresolv $LIB_SOCKET $save_LIBS"
+		[LIBS="-lresolv $LIB_SOCKET $save_LIBS"
 		AC_TRY_LINK([[
 #include <sys/types.h>
 #include <netinet/in.h>
@@ -31,7 +31,7 @@
 res_search( host, C_IN, T_MX, (u_char *)&ans, sizeof(ans));
 return 0;
 ]], LIB_SOCKET="-lresolv $LIB_SOCKET")
-        )
+        ])
 	LIBS="$LIB_SOCKET $save_LIBS"
 	AC_CHECK_FUNCS(dn_expand dns_lookup)
 	LIBS="$save_LIBS"
